layersvt/CMakeLists.txt

```diff
@@ -92,48 +76,31 @@ endif ()
 
 if (WIN32)
     macro(add_vk_layer target)
-        FILE(TO_NATIVE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/VkLayer_${target}.def DEF_FILE)
-        add_custom_target(copy-${target}-def-file ALL
-            COMMAND ${CMAKE_COMMAND} -E copy_if_different ${DEF_FILE} VkLayer_${target}.def
-            VERBATIM
-        )
-        add_library(VkLayer_${target} SHARED ${ARGN} VkLayer_${target}.def)
-        target_link_Libraries(VkLayer_${target} Vulkan::Headers Vulkan::UtilityHeaders Vulkan::LayerSettings ${VkLayer_utils_LIBRARY})
-        set_target_properties(VkLayer_${target} PROPERTIES
-            CXX_STANDARD 17
-            CXX_STANDARD_REQUIRED ON
-            CXX_EXTENSIONS OFF
-        )
-        set_target_properties(copy-${target}-def-file PROPERTIES FOLDER "VkLayer_${target}")
+        add_library(VkLayer_${target} SHARED ${ARGN})
+        if (MSVC)
+            target_link_options(VkLayer_${target} PRIVATE /DEF:${CMAKE_CURRENT_SOURCE_DIR}/VkLayer_${target}.def)
+        elseif(MINGW)
+            target_sources(VkLayer_${target} PRIVATE /DEF:${CMAKE_CURRENT_SOURCE_DIR}/VkLayer_${target}.def)
+        endif()
```

Comment:
Weird that this is different.

Reply:
MINGW doesn't use the MSVC linker it uses a modified version of the GNU linker

Reply:
Also see updated fix what I had there was wrong